tests/Factory/StubFactoryTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\Factory;
+
+
+use Moka\Exception\InvalidArgumentException;
+use Moka\Factory\StubFactory;
+use Moka\Stub\Stub;
+
+class StubFactoryTest extends \PHPUnit_Framework_TestCase
```

Comment:
It's present since 5.4: https://github.com/sebastianbergmann/phpunit/blob/5.4/src/ForwardCompatibility/TestCase.php